index.js

```diff
@@ -1,8 +1,9 @@
-const express = require('express')
-const app = express()
+const express= require('express')
```

Comment:
```suggestion
const express = require('express')
```

It's a standard practice and always good to use a code formatter so that all the formatting is taken care of.

if you are using vscode as code editor, try out prettier extension